server/routes.go

```diff
@@ -1228,6 +1228,7 @@ func (s *Server) GenerateRoutes() http.Handler {
 	r.POST("/api/blobs/:digest", s.CreateBlobHandler)
 	r.HEAD("/api/blobs/:digest", s.HeadBlobHandler)
 	r.GET("/api/ps", s.PsHandler)
+	r.Any("/api/template", gin.WrapF(s.TemplateHandler))
```

Comment:
We always want to support that all HTTP methods a semantically equivalent for this endpoint?

Reply:
Should the endpoint check the method (or maybe later we use the newer built-in routing features?): https://go.dev/blog/routing-enhancements